ftw/keywordwidget/profiles/default/jsregistry.xml

```diff
@@ -2,7 +2,7 @@
 <object name="portal_javascripts">
 
     <javascript
-        id="++resource++ftw.keywordwidget/js/keywordwidget.js"
+        id="++resource++ftw.keywordwidget/js/keywordwidget_4.x.js"
```

Comment:
This needs a upgrade step, which removes the keywordwidget.js and add the other one. 

`<javascript id="++resource++ftw.keywordwidget/js/keywordwidget.js" remove="True" />`

Reply:
I guess I'll just leave the old name like it was...?

Reply:
I renamed the file to its original filename again. Therefore we should not need an upgrade step.